jupyter/datascience/anaconda-python-3.8/setup-elyra.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
```

Comment:
Please match this file https://github.com/opendatahub-io/notebooks/blob/main/jupyter/datascience/ubi8-python-3.8/setup-elyra.sh
to keep consistency.